service/src/main/java/de/adorsys/psd2/sandbox/portal/testdata/TestDataService.java

```diff
@@ -458,11 +494,79 @@ private PsuData initPsuPending() {
         "Amazon.de: Ihre Bestellung #4528499"
     );
 
-    return new PsuData(
+    return new TestPsu(
+        "PSU-Pending",
+        GLOBAL_PASSWORD,
+        GLOBAL_TAN,
+        initSingleAccount(accountId, iban, BigDecimal.valueOf(9.21), transaction),
+        "Pending",
+        "valid",
+        "finalised",
+        null,
+        null,
+        "finalised"
+    );
+  }
+
+  private TestPsu initPsuConsentExpired() {
+    String iban = "DE86760300801729983660";
+    String accountOwner = "Meryl Streep";
```

Comment:
I would use a more neutral and generic name.

Reply:
We will change the names and transactions in a following separate commit.